addons/resource.language.en_gb/resources/strings.po

```diff
@@ -2860,59 +2868,59 @@ msgstr ""
 
 #: xbmc/settings/AdvancedSettings.cpp
 msgctxt "#669"
-msgid "Verbose logging for SMB library"
+msgid "Verbose logging for \"SMB\" library"
 msgstr ""
 
 #: xbmc/settings/AdvancedSettings.cpp
 msgctxt "#670"
-msgid "Verbose logging for CURL library (http, dav)"
+msgid "Verbose logging for \"libcURL\" library \"HTTP(S), DAV\""
 msgstr ""
 
 #empty string with id 671
 
 #: xbmc/settings/AdvancedSettings.cpp
 msgctxt "#672"
-msgid "Verbose logging for FFMPEG libraries"
+msgid "Verbose logging for \"FFmpeg\" libraries"
 msgstr ""
 
 #: xbmc/settings/AdvancedSettings.cpp
 msgctxt "#673"
-msgid "Verbose logging for RTMP library"
+msgid "Verbose logging for \"RTMP\" library"
 msgstr ""
 
 #: xbmc/settings/AdvancedSettings.cpp
 msgctxt "#674"
-msgid "Verbose logging for DBUS calls"
+msgid "Verbose logging for \"D-Bus\" calls"
 msgstr ""
 
 #: xbmc/settings/AdvancedSettings.cpp
 msgctxt "#675"
-msgid "Verbose logging for JSON-RPC requests"
+msgid "Verbose logging for \"JSON-RPC\" requests"
 msgstr ""
 
 #: xbmc/settings/AdvancedSettings.cpp
 msgctxt "#676"
-msgid "Verbose logging for AUDIO component"
+msgid "Verbose logging for \"Audio\" component"
 msgstr ""
 
 #: xbmc/settings/AdvancedSettings.cpp
 msgctxt "#677"
-msgid "Verbose logging for AirTunes library"
+msgid "Verbose logging for \"AirTunes\" library"
 msgstr ""
 
 #: xbmc/settings/AdvancedSettings.cpp
 msgctxt "#678"
-msgid "Verbose logging for UPnP components"
+msgid "Verbose logging for \"UPnP\" components"
 msgstr ""
 
 #: xbmc/settings/AdvancedSettings.cpp
 msgctxt "#679"
-msgid "Verbose logging for CEC library"
+msgid "Verbose logging for \"libCEC\" library"
```

Comment:
`msgid "Verbose logging for \"libCEC\""` without library cause it's double. and since libCEC is the only CEC library and we also name the other libs, I prefer libCEC rather than CEC library.